app/src/main/java/at/gammastrahlung/monopoly_app/activities/BoardActivity.java

```diff
@@ -485,11 +496,72 @@ public void moveAvatarAfterCheating() {
     @Override
     public void onForward(int value) {
         moveAvatar();
+
+        Player currentPlayer = GameData.getGameData().getCurrentPlayer();
+       int currentFieldIndex = currentPlayer.getCurrentFieldIndex();
+        int moveValue = value+currentFieldIndex;
+        followingIndex = (moveValue) % 40;
+        MonopolyClient.getMonopolyClient().sendCurrentFieldInfo(followingIndex);
     }
 
     // Player does want to cheat and moves a selected value forward
     @Override
     public void onSelectedValue(int value){
         moveAvatarAfterCheating();
+        Player currentPlayer = GameData.getGameData().getCurrentPlayer();
```

Comment:
This is the same code as in onForward()

Reply:
Here you need to leave this in because it needs to send the values of the cheating value